daktari/__main__.py

```diff
@@ -30,7 +30,7 @@ def main() -> int:
     os.chdir(args.config_path.parent.absolute())
     print_config_messages(config, args)
 
-    all_passed = run_checks(config.checks, args.quiet_mode or config.quiet_mode, args.fail_fast)
+    all_passed = run_checks(config.checks, args.quiet_mode or config.quiet_mode, args.fail_fast, args.clipboard)
```

Comment:
Sorry to be a pain, but I wonder if it makes sense for this to be a new flag at all?

Passing `--clipboard` automatically also turns on `--fail-fast`. So could clipboard copying just be part of --fail-fast always? I'm struggling to think of a use case where you want `--fail-fast` but would be annoyed by the clipboard behaviour :thinking: